connectors/utils.py

```diff
@@ -73,13 +74,25 @@
     ".vsdm",
 ]
 
+ISO_ZULU_TIMESTAMP_FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
 
 def iso_utc(when=None):
     if when is None:
         when = datetime.now(timezone.utc)
     return when.isoformat()
 
 
+def iso_zulu():
+    """Returns the current time in ISO Zulu format"""
+    return datetime.now(timezone.utc).strftime(ISO_ZULU_TIMESTAMP_FORMAT)
+
+
+def epoch_timestamp():
```

Comment:
```suggestion
def epoch_timestamp_zulu():
```

nit: Just for consistency purposes as the method above also specifies `zulu` explicitly in its method name.